temp/OnnxTransformTempSolution/Microsoft.ML.Transforms.Onnx/OnnxUtils.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Scoring;
+using OnnxShape = System.Collections.Generic.List<long>;
+
+namespace Microsoft.ML.Transforms.Onnx
+{
+    /// <summary>
+    /// OnnxModelInfo contains the data that we should get from 
+    /// Sonoma API once that functionality is added.
+    /// </summary>
+    public sealed class OnnxModelInfo
+    {
+        public OnnxNodeInfo[] InputsInfo;
+        public OnnxNodeInfo[] OutputsInfo;
+    }
+
+    /// <summary>
+    /// OnnxNodeInfo contains all the information for a given node (e.g. inputs/outputs)
+    /// of an Onnx model.
+    /// </summary>
+    public class OnnxNodeInfo
```

Comment:
@shmoradims , this is (roughly) equivalent to the the low-level API Lotus uses. Since we're (most likely) developing a new Lotus# package, it may be better to expose a NodeInfo style API there.

For now, you should be able to use the existing Sonoma API to construct a NodeInfo.